### ISSUE TITLE
[Mocknet] Unresponsive RPC server

### ISSUE THREAD

**Issue by zone117x** [blockstack/stacks-blockchain#1454]
RPC server no longer responsive in testnet with latest master. 

**Steps To Reproduce**
Run `stacks-testnet /app/config/Stacks.toml` with `Stacks.toml`:
```toml
[node]
name = "helium-node"
rpc_bind = "0.0.0.0:9000"

[burnchain]
chain = "bitcoin"
mode = "mocknet"
commit_anchor_block_within = 5000

[[mstx_balance]]
# Private key: b8d99fd45da58038d630d9855d3ca2466e8e0f89d3894c4724f0efc9ff4b51f001
address = "ST2ZRX0K27GW0SP3GJCEMHD95TQGJMKB7G9Y0X1MH"
amount = 10000000000000000

[[mstx_balance]]
# Private key: 3a4e84abb8abe0c1ba37cef4b604e73c82b1fe8d99015cb36b029a65099d373601
address = "ST26FVX16539KKXZKJN098Q08HRX3XBAP541MFS0P"
amount = 10000000000000000

[[mstx_balance]]
# Private key: 052cc5b8f25b1e44a65329244066f76c8057accd5316c889f476d0ea0329632c01
address = "ST3CECAKJ4BH08JYY7W53MC81BYDT4YDA5M7S5F53" 
amount = 10000000000000000

[[mstx_balance]]
# Private key: 9aef533e754663a453984b69d36f109be817e9940519cc84979419e2be00864801
address = "ST31HHVBKYCYQQJ5AQ25ZHA6W2A548ZADDQ6S16GP" 
amount = 10000000000000000
```

Console output:
```
Starting testnet with config /app/config/Stacks.toml...
Transactions can be posted on the endpoint:
POST http://0.0.0.0:9000/v2/transactions
INFO [1587467053.468] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/233c23e/src/chainstate/stacks/db/accounts.rs:179] ST2ZRX0K27GW0SP3GJCEMHD95TQGJMKB7G9Y0X1MH credited: 10000000000000000 uSTX
INFO [1587467053.473] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/233c23e/src/chainstate/stacks/db/accounts.rs:179] ST26FVX16539KKXZKJN098Q08HRX3XBAP541MFS0P credited: 10000000000000000 uSTX
INFO [1587467053.476] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/233c23e/src/chainstate/stacks/db/accounts.rs:179] ST3CECAKJ4BH08JYY7W53MC81BYDT4YDA5M7S5F53 credited: 10000000000000000 uSTX
INFO [1587467053.481] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/233c23e/src/chainstate/stacks/db/accounts.rs:179] ST31HHVBKYCYQQJ5AQ25ZHA6W2A548ZADDQ6S16GP credited: 10000000000000000 uSTX
INFO [1587467053.515] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/233c23e/testnet/src/event_dispatcher.rs:180] Registering event observer at: host.docker.internal:3700
BOOTSTRAP WITH []
INFO [1587467053.628] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/233c23e/testnet/src/node.rs:259] Bound HTTP server on: 0.0.0.0:9000
INFO [1587467053.628] [/usr/local/cargo/git/checkouts/stacks-blockchain-e574f41bf84d9c41/233c23e/testnet/src/node.rs:260] Bound P2P server on: 127.0.0.1:62890
```

Run `curl http://127.0.0.1:9000/v2/info` or open in web browser.

Request waits indefinitely.

**Comment by lgalabru** [comment]
Addressed with #1456

**Comment by zone117x** [comment]
Fixed thanks @lgalabru you rock
